cmd/up/up.go

```diff
@@ -789,7 +791,12 @@ func (up *upContext) waitUntilExitOrInterruptOrApply(ctx context.Context) error
 		case err := <-up.GlobalForwarderStatus:
 			oktetoLog.Infof("exiting by error in global forward checker: %v", err)
 			return err
-
+		case <-ttySignals:
+			oktetoLog.Infof("SIGTTOU/SIGTTIN received, starting shutdown sequence preventing it to background the process")
+			return oktetoErrors.UserError{
+				E:    fmt.Errorf("terminal disconnect detected"),
+				Hint: "Try re-running 'okteto up' to reconnect",
```

Comment:
Here's my suggestion, when the CLI disconnects we display a warning: `Connection lost to your development container, reconnecting...` in this case we cannot reconnect, so it's an error, but with a similar message.

```suggestion
				E:    fmt.Errorf("connection lost to your development container"),
				Hint: "Use 'okteto up' to reconnect",
```